---
title: ""
author: "Jodie Lord"
date: "12/08/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

_______________________________________________________________________


# Publications

_______________________________________________________________________

- ###  [Integrated lipidomics and proteomics network analysis highlights lipid and immunity pathways associated with Alzheimer’s disease](https://link.springer.com/epdf/10.1186/s40035-020-00215-0?sharing_token=rDhcqrDwxHpPr9KVQnNAQm_BpE1tBhCbnbw3BuzI2RMack7kbujqP_X-ofWsx-46ldfN8uNgddF6WQJfINxSB6Wsva1Vqwae_tqTdQl2TrKHAl08e7-qKVWzx4Ae986dEAZF2P308PiyisWZpKP16t2NcjKBdygtKOfwRVD1Nno%3D) 
<br/>
_Jin Xu, Giulia Bankov, Min Kim, Asger Wretlind, **Jodie Lord**, Rebecca Green, Angela Hodges, Abdul Hye, Dag Aarsland, Latha Velayudhan, Richard J.B. Dobson, Petroula Proitsi, Cristina Legido-Quigley, on behalf of the AddNeuroMed Consortium (2020). Translational Neurodegeneration. 9,1, p.36_
<br/>
<br/>
Background: There is an urgent need to understand the pathways and processes underlying Alzheimer’s disease (AD) for early diagnosis and development of effective treatments. This study was aimed to investigate Alzheimer’s dementia using an unsupervised lipid, protein and gene multi-omics integrative approach. Methods: A lipidomics dataset comprising 185 AD patients, 40 mild cognitive impairment (MCI) individuals and 185 controls, and two proteomics datasets (295 AD, 159 MCI and 197 controls) were used for weighted gene coexpression network analyses (WGCNA). Correlations of modules created within each modality with clinical AD diagnosis, brain atrophy measures and disease progression, as well as their correlations with each other, were analyzed. Gene ontology enrichment analysis was employed to examine the biological processes and molecular and cellular functions of protein modules associated with AD phenotypes. Lipid species were annotated in the lipid modules associated with AD phenotypes. The associations between established AD risk loci and the lipid/protein modules that showed high correlation with AD phenotypes were also explored. Results: Five of the 20 identified lipid modules and five of the 17 identified protein modules were correlated with clinical AD diagnosis, brain atrophy measures and disease progression. The lipid modules comprising phospholipids, triglycerides, sphingolipids and cholesterol esters were correlated with AD risk loci involved in immune response and lipid metabolism. The five protein modules involved in positive regulation of cytokine production, neutrophilmediated immunity, and humoral immune responses were correlated with AD risk loci involved in immune and complement systems and in lipid metabolism (the APOE ε4 genotype).
<br/>
<br/>

_______________________________________________________________________


# Open Access Pre-prints

_______________________________________________________________________

<br/>

- ### [Deciphering the causal relationship between blood metabolites and Alzheimers Disease: a Mendelian Randomization study](https://www.medrxiv.org/content/10.1101/2020.04.28.20083253v1) 
<br/>
_**Jodie Lord**, Bradley Jermy, Rebecca Green, Andrew Wong, Jin Xu, Cristina Legido-Quigley, Richard Dobson, Marcus Richards, Petra Proitsi (2020)_
<br/>
<br/>
BACKGROUND: There are currently no disease modifying treatments for Alzheimers Disease (AD). Epidemiological studies have highlighted blood metabolites as potential biomarkers, but possible confounding and reverse causation prevent causal conclusions. Here, we investigated whether nineteen metabolites previously associated with midlife cognitive function, are on the causal pathway to AD. METHODS: Summary statistics from the largest Genome-Wide Association Studies (GWAS) for AD and for metabolites were used to perform bi-directional univariable Mendelian Randomisation (MR). Bayesian model averaging MR (MR-BMA) was additionally performed to address high correlation between metabolites and to identify metabolite combinations which may be on the AD causal pathway. RESULTS: Univariable MR indicated three Extra-Large High-Density Lipoproteins (XL.HDL) to be on the causal pathway to AD: Free Cholesterol (XL.HDL.FC: OR=0.86, 95% CI=0.78-0.94), Total Lipids (XL.HDL.L: OR=0.88, 95% CI=0.80-0.97), and Phospholipids (XL.HDL.PL: OR=0.87, 95% CI=0.81-0.97); significant at an adjusted threshold of p<0.009. MR-BMA corroborated XL.HDL.FC to be amongst the top three causal metabolites, additionally to Total Cholesterol in XL.HDL (XL.HDL.C) and Glycoprotein Acetyls (GP) (posterior probabilities=0.112, 0.113, 0.287 respectively). Both XL.HDL.C and GP also demonstrated suggestive evidence of univariable causal associations (XL.HDL.C:OR=0.88, 95% CI=0.79-0.99; GP:OR=1.2, 95% CI=1.05-1.38); significant at the 5% level. DISCUSSION: This study offers insight into the causal relationship between metabolites previously demonstrating association with mid-life cognition, and AD. It highlights GP in addition to several XL.HDLs as causal candidates which warrant further investigation. As the pathological changes underpinning AD are thought to develop decades prior to symptom onset, progressing these findings could hold special value in informing future risk reduction strategies.
<br/>
<br/>


- ###  [Integrated lipidomics and proteomics network analysis highlights lipid and immunity pathways associated with Alzheimer’s disease](https://www.biorxiv.org/content/10.1101/2020.03.18.995464v2) 
<br/>
_Jin Xu, Giulia Bankov, Min Kim, Asger Wretlind, **Jodie Lord**, Rebecca Green, Angela Hodges, Abdul Hye, Dag Aarsland, Latha Velayudhan, Richard J.B. Dobson, Petroula Proitsi, Cristina Legido-Quigley, on behalf of the AddNeuroMed Consortium (2020)_
<br/>
<br/>
BACKGROUND: There is an urgent need to understand the molecular mechanisms underlying Alzheimer’s Disease (AD) to enable early diagnosis and develop effective treatments. Here we aim to investigate Alzheimer’s dementia using an unsupervised lipid, protein and gene multi-omic integrative approach. METHODS: A lipidomics dataset (185 AD, 40 MCI and 185 controls) and a proteomics dataset (201 AD patients, 104 MCI individuals and 97 controls) were utilised for weighted gene co-expression network analyses (WGCNA). An additional proteomics dataset (94 AD, 55 MCI and 100 controls) was included for external proteomics validation. Modules created within each modality were correlated with clinical AD diagnosis, brain atrophy measures and disease progression, as well as with each other. Gene Ontology (GO) enrichment analysis was employed to examine the biological processes and molecular and cellular functions for protein modules associated with AD phenotypes. Lipid species were annotated in the lipid modules associated with AD phenotypes. Associations between established AD risk loci and lipid/protein modules that showed high correlation with AD phenotypes were also explored. RESULTS: Five of the 20 identified lipid modules and five of the 17 identified protein modules were correlated with AD phenotypes. Lipid modules comprising of phospholipids, triglycerides, sphingolipids and cholesterol esters, correlated with AD risk loci involved in immune response and lipid metabolism. Five protein modules involved in positive regulation of cytokine production, neutrophil mediated immunity, humoral immune responses were correlated with AD risk loci involved in immune and complement systems. DISCUSSION: We have shown the first multi-omic study linking genes, proteins and lipids to study pathway dysregulation in AD. Results identified modules of tightly regulated lipids and proteins that were strongly associated with AD phenotypes and could be pathology drivers in lipid homeostasis and innate immunity.
<br/>
