---
title: ""
author: "Jodie Lord"
date: "12/08/2020"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

_______________________________________________________________________


# Publications

_______________________________________________________________________

<br/>

### [Association between polygenic risk score of Alzheimer's disease and plasma phosphorylated tau in individuals from the Alzheimer's Disease Neuroimaging Initiative](https://pubmed.ncbi.nlm.nih.gov/33419453/) 
<br/>
Anna Zettergren, **Jodie Lord**, Nicholas J Ashton, Andrea L Benedet, Thomas K Karikari, Juan Lantero Rodriguez, Alzheimer’s Disease Neuroimaging Initiative; Anniina Snellman, Marc Suárez-Calvet, Petroula Proitsi, Henrik Zetterberg, Kaj Blennow (2021)
<br/>
<br/>
**BACKGROUND:** Recent studies suggest that plasma phosphorylated tau181 (p-tau181) is a highly specific biomarker for Alzheimer's disease (AD)-related tau pathology. It has great potential for the diagnostic and prognostic evaluation of AD, since it identifies AD with the same accuracy as tau PET and CSF p-tau181 and predicts the development of AD dementia in cognitively unimpaired (CU) individuals and in those with mild cognitive impairment (MCI). Plasma p-tau181 may also be used as a biomarker in studies exploring disease pathogenesis, such as genetic or environmental risk factors for AD-type tau pathology. The aim of the present study was to investigate the relation between polygenic risk scores (PRSs) for AD and plasma p-tau181. **METHODS:** Data from the Alzheimer's Disease Neuroimaging Initiative (ADNI) was used to examine the relation between AD PRSs, constructed based on findings in recent genome-wide association studies, and plasma p-tau181, using linear regression models. Analyses were performed in the total sample (n = 818), after stratification on diagnostic status (CU (n = 236), MCI (n = 434), AD dementia (n = 148)), and after stratification on Aβ pathology status (Aβ positives (n = 322), Aβ negatives (n = 409)). **RESULTS:** Associations between plasma p-tau181 and APOE PRSs (p = 3e-18-7e-15) and non-APOE PRSs (p = 3e-4-0.03) were seen in the total sample. The APOE PRSs were associated with plasma p-tau181 in all diagnostic groups (CU, MCI, and AD dementia), while the non-APOE PRSs were associated only in the MCI group. The APOE PRSs showed similar results in amyloid-β (Aβ)-positive and negative individuals (p = 5e-5-1e-3), while the non-APOE PRSs were associated with plasma p-tau181 in Aβ positives only (p = 0.02). **CONCLUSIONS:** Polygenic risk for AD including APOE was found to associate with plasma p-tau181 independent of diagnostic and Aβ pathology status, while polygenic risk for AD beyond APOE was associated with plasma p-tau181 only in MCI and Aβ-positive individuals. These results extend the knowledge about the relation between genetic risk for AD and p-tau181, and further support the usefulness of plasma p-tau181 as a biomarker of AD.
<br/>
<br/>

###  [Integrated lipidomics and proteomics network analysis highlights lipid and immunity pathways associated with Alzheimer’s disease](https://link.springer.com/epdf/10.1186/s40035-020-00215-0?sharing_token=rDhcqrDwxHpPr9KVQnNAQm_BpE1tBhCbnbw3BuzI2RMack7kbujqP_X-ofWsx-46ldfN8uNgddF6WQJfINxSB6Wsva1Vqwae_tqTdQl2TrKHAl08e7-qKVWzx4Ae986dEAZF2P308PiyisWZpKP16t2NcjKBdygtKOfwRVD1Nno%3D) 
<br/>
Jin Xu, Giulia Bankov, Min Kim, Asger Wretlind, **Jodie Lord**, Rebecca Green, Angela Hodges, Abdul Hye, Dag Aarsland, Latha Velayudhan, Richard J.B. Dobson, Petroula Proitsi, Cristina Legido-Quigley, on behalf of the AddNeuroMed Consortium (2020). Translational Neurodegeneration. 9,1, p.36
<br/>
<br/>
**Background:** There is an urgent need to understand the pathways and processes underlying Alzheimer’s disease (AD) for early diagnosis and development of effective treatments. This study was aimed to investigate Alzheimer’s dementia using an unsupervised lipid, protein and gene multi-omics integrative approach. **Methods:** A lipidomics dataset comprising 185 AD patients, 40 mild cognitive impairment (MCI) individuals and 185 controls, and two proteomics datasets (295 AD, 159 MCI and 197 controls) were used for weighted gene coexpression network analyses (WGCNA). Correlations of modules created within each modality with clinical AD diagnosis, brain atrophy measures and disease progression, as well as their correlations with each other, were analyzed. Gene ontology enrichment analysis was employed to examine the biological processes and molecular and cellular functions of protein modules associated with AD phenotypes. Lipid species were annotated in the lipid modules associated with AD phenotypes. The associations between established AD risk loci and the lipid/protein modules that showed high correlation with AD phenotypes were also explored. **Results:** Five of the 20 identified lipid modules and five of the 17 identified protein modules were correlated with clinical AD diagnosis, brain atrophy measures and disease progression. The lipid modules comprising phospholipids, triglycerides, sphingolipids and cholesterol esters were correlated with AD risk loci involved in immune response and lipid metabolism. The five protein modules involved in positive regulation of cytokine production, neutrophilmediated immunity, and humoral immune responses were correlated with AD risk loci involved in immune and complement systems and in lipid metabolism (the APOE ε4 genotype).
<br/>
<br/>

_______________________________________________________________________


# Open Access Pre-prints

_______________________________________________________________________

<br/>

### [Deciphering the causal relationship between blood metabolites and Alzheimers Disease: a Mendelian Randomization study](https://www.medrxiv.org/content/10.1101/2020.04.28.20083253v1) 
<br/>
**Jodie Lord**, Bradley Jermy, Rebecca Green, Andrew Wong, Jin Xu, Cristina Legido-Quigley, Richard Dobson, Marcus Richards, Petra Proitsi (2020)
<br/>
<br/>
**BACKGROUND:** There are currently no disease modifying treatments for Alzheimers Disease (AD). Epidemiological studies have highlighted blood metabolites as potential biomarkers, but possible confounding and reverse causation prevent causal conclusions. Here, we investigated whether nineteen metabolites previously associated with midlife cognitive function, are on the causal pathway to AD. **METHODS:** Summary statistics from the largest Genome-Wide Association Studies (GWAS) for AD and for metabolites were used to perform bi-directional univariable Mendelian Randomisation (MR). Bayesian model averaging MR (MR-BMA) was additionally performed to address high correlation between metabolites and to identify metabolite combinations which may be on the AD causal pathway. **RESULTS:** Univariable MR indicated three Extra-Large High-Density Lipoproteins (XL.HDL) to be on the causal pathway to AD: Free Cholesterol (XL.HDL.FC: OR=0.86, 95% CI=0.78-0.94), Total Lipids (XL.HDL.L: OR=0.88, 95% CI=0.80-0.97), and Phospholipids (XL.HDL.PL: OR=0.87, 95% CI=0.81-0.97); significant at an adjusted threshold of p<0.009. MR-BMA corroborated XL.HDL.FC to be amongst the top three causal metabolites, additionally to Total Cholesterol in XL.HDL (XL.HDL.C) and Glycoprotein Acetyls (GP) (posterior probabilities=0.112, 0.113, 0.287 respectively). Both XL.HDL.C and GP also demonstrated suggestive evidence of univariable causal associations (XL.HDL.C:OR=0.88, 95% CI=0.79-0.99; GP:OR=1.2, 95% CI=1.05-1.38); significant at the 5% level. **DISCUSSION:** This study offers insight into the causal relationship between metabolites previously demonstrating association with mid-life cognition, and AD. It highlights GP in addition to several XL.HDLs as causal candidates which warrant further investigation. As the pathological changes underpinning AD are thought to develop decades prior to symptom onset, progressing these findings could hold special value in informing future risk reduction strategies.
<br/>
<br/>


###  [Metabolic correlates of late midlife cognitive function: findings from the 1946 British Birth Cohort](https://www.medrxiv.org/content/10.1101/2020.11.23.20236463v1.full.pdf) 
<br/>
Rebecca Green, **Jodie Lord**, Jin Xu, Jane Maddock, Min Kim, Richard Dobson, Cristina Legido-Quigley, Andrew Wong, Marcus Richards, Petroula Proitsi
<br/>
<br/>
**Background** Investigating associations between metabolites and late midlife cognitive function could reveal potential markers and mechanistic insights relevant to early dementia. Here, we aimed to identify the metabolic underpinnings of cognitive outcomes in late midlife by exploring and integrating associations of single metabolites, metabolic pathways and networks. We further aimed to untangle the influence of life course factors on these relationships; a previously unexplored avenue using a systems biology approach. **Methods and Findings** Levels of 1019 metabolites were detected by liquid chromatography-mass spectrometry (Metabolon Inc) and quantified at age 60-64 among participants of the British 1946 Birth Cohort (N=1740). Cognitive outcomes were assessed at the same age and 5-9 years later, and included short-term memory (age 60-64, 69 and change), delayed memory (age 60-64), processing speed (age 60-64, 69 and change) and Addenbrooke’s Cognitive Examination III (age 69). Using a combination of linear regression analysis, quantitative pathway analysis and weighted gene correlation network analysis, we evaluated relationships between metabolite measures (single-metabolites, pathways and network modules) and cognitive outcomes. Single-metabolite and network analyses were sequentially adjusted for life course factors across four models, including: sex and blood clinic information (model 1); model 1 + BMI and lipid medication (model 2); model 2 + childhood cognition, education and socioeconomic position (model 3); model 3 + smoking, exercise, alcohol intake, blood pressure and diet (model 4). After correcting for multiple tests, we identified 155 metabolites, 10 pathways and 5 modules to show relationships with cognitive outcomes. Thirty-five metabolites were influential in their module and identified in single-metabolite analyses. Notably, we report independent relationships between a module comprised of acylcarnitines and processing speed, revealing palmitoylcarnitine (C16) as a key driver of associations (model 4: ß = −0.10, 95%CI = −0.15 to −0.052). Two modules demonstrated associations with several cognitive outcomes that were partly explained by life course factors: one enriched in modified nucleosides and amino acids (ß range (model 1) = −0.12 to −0.09, attenuation (model 4)= 39.2 to 55.5%), and another in vitamin A and C metabolites (ß range (model 1) = 0.11 to 0.23, attenuation (model 4) = 68.6 to 92.6%). Our other findings, including a module enriched in sphingolipid pathways (ß range (model 1) = 0.085 to 0.10, attenuation (model 4) = 87.0 to 116%), were entirely explained by life course factors particularly childhood cognition and education. The limitations of this study include those commonly seen with population-based cohorts, such as possible residual confounding and generalisability to other populations, as well as a lack of longitudinal metabolite data. **Conclusions** Using a large birth cohort study with information across the life course, we highlighted potential metabolic mechanisms underlying cognitive function in late midlife, suggesting marker candidates and life course relationships for further study.
<br/>
<br/>


###  [Integrated lipidomics and proteomics network analysis highlights lipid and immunity pathways associated with Alzheimer’s disease](https://www.biorxiv.org/content/10.1101/2020.03.18.995464v2) 
<br/>
Jin Xu, Giulia Bankov, Min Kim, Asger Wretlind, **Jodie Lord**, Rebecca Green, Angela Hodges, Abdul Hye, Dag Aarsland, Latha Velayudhan, Richard J.B. Dobson, Petroula Proitsi, Cristina Legido-Quigley, on behalf of the AddNeuroMed Consortium (2020)
<br/>
<br/>
BACKGROUND: There is an urgent need to understand the molecular mechanisms underlying Alzheimer’s Disease (AD) to enable early diagnosis and develop effective treatments. Here we aim to investigate Alzheimer’s dementia using an unsupervised lipid, protein and gene multi-omic integrative approach. METHODS: A lipidomics dataset (185 AD, 40 MCI and 185 controls) and a proteomics dataset (201 AD patients, 104 MCI individuals and 97 controls) were utilised for weighted gene co-expression network analyses (WGCNA). An additional proteomics dataset (94 AD, 55 MCI and 100 controls) was included for external proteomics validation. Modules created within each modality were correlated with clinical AD diagnosis, brain atrophy measures and disease progression, as well as with each other. Gene Ontology (GO) enrichment analysis was employed to examine the biological processes and molecular and cellular functions for protein modules associated with AD phenotypes. Lipid species were annotated in the lipid modules associated with AD phenotypes. Associations between established AD risk loci and lipid/protein modules that showed high correlation with AD phenotypes were also explored. RESULTS: Five of the 20 identified lipid modules and five of the 17 identified protein modules were correlated with AD phenotypes. Lipid modules comprising of phospholipids, triglycerides, sphingolipids and cholesterol esters, correlated with AD risk loci involved in immune response and lipid metabolism. Five protein modules involved in positive regulation of cytokine production, neutrophil mediated immunity, humoral immune responses were correlated with AD risk loci involved in immune and complement systems. DISCUSSION: We have shown the first multi-omic study linking genes, proteins and lipids to study pathway dysregulation in AD. Results identified modules of tightly regulated lipids and proteins that were strongly associated with AD phenotypes and could be pathology drivers in lipid homeostasis and innate immunity.
<br/>
